### ISSUE TITLE
function parameters naming convention

### ISSUE THREAD

**Issue by markusadamek** [fooof-tools/fooof_mat#16]
Hi, 
in the python implementation the setting to adjust the aperiodic fit is called 'aperiodic_mode', in the matlab wrapper however, the setting is called 'background_mode'.

This is might cause unnecessary confusion (at least it did for me) and it might be better to rename the background mode to aperiodic_mode for consistency

**Comment by TomDonoghue** [comment]
Yes - you are completely correct, sorry about that. We have not been super clear on noting some of the changes going on. 

The reason there is the difference is that we are in the midst of a major release, that has some name changes - this reflects the change in name in the original Python version. However, that is not properly released yet, so if you download python-FOOOF with pip, it actually will still have the old name (background_mode, etc) and so will work with the wrapper right now. 

Long story short:
- current version here should work if you use the pip installed FOOOF (though this is different from what is currently on Github)
- if you want to use the newest version, you can install the Python version from Github, and install the fooof_mat wrapper from the 'dev' branch here, and everything is updated. 

It should all be merged / released and become consistent soon! Thanks for trying it out!

**Comment by TomDonoghue** [comment]
New releases have been made, so the current master branch of this repository is now up to date with the new release of the Python FOOOF version.
